Resources/Locale/en-US/paper/stamp-component.ftl

```diff
@@ -15,3 +15,5 @@ stamp-component-stamped-name-warden = Warden
 stamp-component-stamped-name-trader = Trader
 stamp-component-stamped-name-syndicate = Syndicate
 stamp-component-stamped-name-ce = Chief Engineer
+stamp-component-stamped-name-lawyer = Lawyer
+stamp-component-stamped-name-psychologist = Psychologist
```

Comment:
We don't need to modify upstream localization files. 

These two lines need to be moved to a Resources/Locale/en-US/DeltaV/paper/ folder(You'll have to make a folder for this). You can basically copy the stamp-component.ftl file there, and delete everything but the lawyer and psychologist localizations.

Reply:
forgot we had our own locale folder as well